Samples/SendReceiveBatch/Sender.cs

```diff
@@ -0,0 +1,82 @@
+using AzureBatchQueue;
+using AzureBatchQueue.Utils;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace SendReceiveBatch;
+
+public class Sender
+{
+    private BatchQueue<string> batchQueue;
+    private const int maxBatchSizeInBytes = 30;
+    private IMessageBatchSerializer<string> serializer;
+
+    public Sender(string queueName, ILogger<BatchQueue<string>> logger, IMessageBatchSerializer<string> serializer)
+    {
+        this.serializer = serializer;
+        batchQueue = new BatchQueue<string>("UseDevelopmentStorage=true", queueName, flushPeriod: TimeSpan.FromSeconds(5), logger: logger);
+    }
+
+    public async Task Init()
+    {
+        await batchQueue.Init();
+        await batchQueue.ClearMessages();
+        Log("Sender is ready.");
+    }
+
+    public async Task SendMessages()
+    {
+        var words = new[] { "One", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten" };
+
+        var batches = CreateBatches(words);
+
+        foreach (var batch in batches)
+        {
+            await batchQueue.SendBatch(batch);
+
+            Log($"Sent a batch {JsonConvert.SerializeObject(batch.Items())} of size {batch.GetBatchSizeInBytes()} bytes.");
+            Sleep(TimeSpan.FromSeconds(5));
+        }
+    }
+
+    static void Sleep(TimeSpan sleep)
+    {
+        Log($"Sleep for {sleep.Seconds} seconds.");
+        Thread.Sleep(sleep);
+    }
+
+    private IEnumerable<MessageBatch<string>> CreateBatches(IReadOnlyList<string>? words)
```

Comment:
What do you think about making MessageBatch an implementation detail and not exposing it publicly?
I mean, having client code would look more like this:

```csharp
queue.Send(items); // will make as many batches as needed and enqueue them.
```

Reply:
Wondering about having a bathcing-related `Sender`/`Reciever` code under the hood of BatchQueue. As a user, I'm interested in sending/reading items from the queue. But the way these items are batched seems like an implementation detail that may hidden inside. Keeping the queue interface as close to the original one (Azure QueueClient) as possible could simplify its integration